packages/blockchain-wallet-v4-frontend/src/modals/SendBitcoin/FirstStep/template.success.js

```diff
@@ -27,12 +27,8 @@ const ColLeft = styled.div`
   align-items: flex-start;
   width: 50%;
 `
-const ColRight = ColLeft.extend`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
+const ColRight = styled(ColLeft)`
```

Comment:
The naming is probably not ideal here

Reply:
Yea well it has been there for ages, agree the naming is not great ! it should only be called Column maybe and pass a specific prop to change the alignment. Will do another time !